packages/shared/src/components/containers/FeedTopicCard.tsx

```diff
@@ -1,53 +1,93 @@
 import classNames from 'classnames';
-import React, { ReactElement, useContext } from 'react';
-import FeaturesContext from '../../contexts/FeaturesContext';
+import React, { ReactElement, ReactNode } from 'react';
 import { TagCategory } from '../../graphql/feedSettings';
 import classed from '../../lib/classed';
+import { OnboardingFiltersLayout } from '../../lib/featureValues';
+import PlusIcon from '../icons/Plus';
 import VIcon from '../icons/V';
 
 interface FeedTopicCardProps {
   topic: TagCategory;
   isActive?: boolean;
+  topicLayout?: OnboardingFiltersLayout;
   onClick?: () => void;
 }
 
+interface ClassName {
+  container?: string;
+  check?: string;
+  text?: string;
+}
+
 const BackgroundLayer = classed(
   'div',
   'group-hover:z-0 absolute inset-0 w-full h-full rounded-14 bg-theme-status-cabbage transition-transform',
 );
 
+const background: Record<OnboardingFiltersLayout, ReactNode> = {
+  grid: (
```

Comment:
Did we not choose to always go for the Enums as well?
It will make it way easier to find and replace if they ever need changing?

But maybe I'm wrong.

Reply:
I remember we decided to strictly use `enum` as `key` whenever we don't use `Record` as with this type, changes and missing keys will surface as well.